docs/pages/concepts/architecture/architecture.md

```diff
@@ -22,8 +22,8 @@ Meshery and its components are written using the following languages and technol
 | Meshery Provider UI                                                  | ReactJS, NextJS                                                                   |
 | [Meshery Remote Providers](/extensibility/providers)                 | _any_ - must adhere to Meshery [Extension Points]({{site.baseurl}}/extensibility) |
 | [Meshery Operator](/concepts/architecture/operator)                  | Golang                                                                            |
-| &nbsp;&nbsp;&nbsp;&nbsp; [MeshSync](/concepts/architecture/meshsync) | Golang                                                                            |
-| &nbsp;&nbsp;&nbsp;&nbsp; [Broker](/concepts/architecture/broker)     | Golang, NATS                                                                      |
```

Comment:
These are sub-components and meant to be indented.

Reply:
Uh oh. The issue asked me to do it, so I went ahead and did it hahah. Sorry, I should have thought this through!